server/src/main/java/org/apache/cloudstack/outofbandmanagement/OutOfBandManagementServiceImpl.java

```diff
@@ -578,7 +578,7 @@ protected void runInContext() {
 
         @Override
         public Long getDelay() {
-            return null;
+            return OutOfBandManagementBackgroundTaskExecutionInterval.value() * 1000L;
```

Comment:
When this returns null, I think the poll manager uses some default value (I think 5s?). If the default config value is 4s, it may not make much difference.

Reply:
@rhtyd The default value when it returns null, in poll manager, is 4s. The proposal of this changes is to externalize the configuration of OOBM interval to the operators, but keeping the same behavior as before if it is not changed.